style.css

```
@@ -0,0 +1,49 @@
+* {
+    font-family: sans-serif;
+}
+
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+    background: #0a0a23;
+}
+
+.container {
+    position: relative;
+    max-width: 600px;
+    height: 500px;
+    padding: 0px;
+    color: black;
+    text-align: center;
+}
+
+.language-section {
+    background: #ffffff;
+}
+
+.button {
+    text-transform: uppercase;
+    text-decoration: none;
+    padding: 0 15px;
+    font-weight: bold;
+    background: #feac32;
+    color: #0a0a23;
```

This varies a lot from team to team, I prefer caps, but as long as the codebase is consistent, that's what matters.
```suggestion
    background: #FEAC32;
    color: #0A0A23;
```